lti/sentry.py

```diff
@@ -0,0 +1,19 @@
+"""Report exceptions to Sentry."""
+
+import os
+
+import raven
+
+
+def get_raven_client(request):
+    """Return the Raven client for reporting crashes to Sentry."""
+    client = request.registry["raven.client"]
+    client.http_context({"url": request.url, "method": request.method})
+    request.add_finished_callback(lambda request: client.context.clear())
+    return client
+
+
+def includeme(config):
+    environment = os.environ.get('ENV', 'dev')
+    config.registry["raven.client"] = raven.Client(environment=environment)
```

Comment:
We need to set the environment here as well, which should be a new environment variable setting similar to h (https://github.com/hypothesis/h/blob/master/h/sentry.py#L52).

That's because we only have one sentry project for both prod and qa and filter the issues based on the environment.